### PR TITLE
data: mdarslan7

### PR DESCRIPTION
**Pull request by mdarslan7**
## Changes proposed

Started contributing to Open Source!

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5359"></a>

### CODE REVIEW

File: data/mdarslan7.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Md Arslan",
+  "type": "personal",
+  "bio": "I am Arslan, a tech geek & nerd who loves programming and philosophy. I am an Open Source & Web3 Enthusiast.",
+
```
Comment (schmelto) [inline data/mdarslan7.json:5]:
```suggestion
```

File: data/mdarslan7.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  "tags": ["Blockchain", "Open Source", "Web Dev"],
+
```
Comment (schmelto) [inline data/mdarslan7.json:7]:
```suggestion
```

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Comment by SaraJaoude** [comment]
looks good

**Pull request merged**